Small fixes to the docs (#36)
nextflow-io/nf-validation: docs/sample_sheet_schema/schema_specification.md

```

If your sample sheet has a different format (for example, a YAML file with a nested structure),

> you will need to build your schema to match the parsed structure. Using nested YAML files is (for now) not officially supported by the plugin and can result in unexpected errors.

Reviewer: Didn't we just say that it was supported, a few lines above?

Author: Good point I'll remove that!

```

will convert the `field` value to a meta value, resulting in the channel `[[id:value, sample:value]...]`

> See [here](https://github.com/nextflow-io/nf-validation/blob/master/plugins/nf-validation/src/testResources/schema_input.json#L10-L25) for an example in the sample sheet.

Reviewer: Should use permalinks:

```suggestion
See [here](https://github.com/nextflow-io/nf-validation/blob/ce3aef60e5103ea4798375fe6c59bae41b7d2a25/plugins/nf-validation/src/testResources/schema_input.json#L10-L25) for an example in the sample sheet.
```

Reviewer: <img width="366" alt="CleanShot 2023-05-31 at 09 43 12@2x" src="https://github.com/nextflow-io/nf-validation/assets/465550/4c815145-4a28-45ce-b711-d4349401af92">

Default: `false`

> - Can be `true`, in which case all fields in this column should be unique for all samples.

Reviewer: I think throughout most of this documentation, fields = columns.

Do you mean rows, or samples?

```suggestion
- Can be `true`, in which case the value for this field should be unique for all samples in the sample sheet.
```

Author: Good point, thanks for the consistency in definitions here :)

    * `field1` isn't unique since `value1` has been found more than once.
    * `field2` isn't unique in combination with `field1` because the `value1,value2` combination has been found more than once.

>     See [`schema_input.json#L48-L55`](https://github.com/nextflow-io/nf-validation/blob/master/plugins/nf-validation/src/testResources/schema_input.json#L48-L55)

Reviewer: ```suggestion
    See [`schema_input.json#L48-L55`](https://github.com/nextflow-io/nf-validation/blob/ce3aef60e5103ea4798375fe6c59bae41b7d2a25/plugins/nf-validation/src/testResources/schema_input.json#L48-L55)
```

    - [ ] The second row will fail because `field1` is set, but `field2` isn't and `field1` is dependent on `field2`.
    - [x] The third row will pass the check because `field1` isn't set.

>     See [here](https://github.com/nextflow-io/nf-validation/blob/master/plugins/nf-validation/src/testResources/schema_input.json#L10-L25) for an example in the sample sheet.

Reviewer: ```suggestion
    See [here](https://github.com/nextflow-io/nf-validation/blob/ce3aef60e5103ea4798375fe6c59bae41b7d2a25/plugins/nf-validation/src/testResources/schema_input.json#L10-L25) for an example in the sample sheet.
```
